tests/runtime/btf

```diff
@@ -45,6 +45,16 @@ REQUIRES lsmod | grep '^nf_tables'
 REQUIRES /usr/sbin/nft --help
 CLEANUP nft delete table bpftrace
 
+NAME kernel_module_attach_wildcard
```

Comment:
Would be neat to have a `NEW_NETNS` runtime test directive to provide a little more isolation. But not needed for now